featuretools/tests/entityset_tests/test_entity.py

```diff
@@ -275,3 +276,30 @@ def test_replace_latlong_nan_during_entity_creation(pd_es):
     with pytest.warns(UserWarning, match="LatLong columns should contain only tuples. All single 'NaN' values in column 'latlong' have been replaced with '\\(NaN, NaN\\)'."):
         entity = ft.Entity(id="nan_latlong_entity", df=df, entityset=nan_es, variable_types=pd_es['log'].variable_types)
     assert entity.df['latlong'][0] == (np.nan, np.nan)
+
+
+def test_text_deprecation_warning():
+    data = pd.DataFrame({
+        "id": [1, 2, 3, 4, 5],
+        "value": ["a", "c", "b", "a", "a"]
+    })
+
+    for text_repr in ['text', ft.variable_types.Text]:
+        es = ft.EntitySet()
+        warnings.resetwarnings()
+        with warnings.catch_warnings(record=True) as ws:
```

Comment:
we could use `with pytest.warns` for these warnings checks

Reply:
Was not aware of this function. Fixed.